src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md

```diff
@@ -1,17 +1,53 @@
-# OpenTelemetry.Instrumentation.GrpcNetClient
+# Grpc.Net.Client Instrumentation for OpenTelemetry
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.GrpcNetClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.GrpcNetClient)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.GrpcNetClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.GrpcNetClient)
 
-Automatically instruments the outgoing requests from
-[Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client).
+This is an [Instrumentation Library](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/glossary.md#instrumentation-library)
+which instruments [Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client)
+and collects telemetry about outgoing gRPC requests.
 
-## Installation
+## Steps to enable OpenTelemetry.Instrumentation.GrpcNetClient
+
+### Step 1: Install Package
+
+Add a reference to the
+[`OpenTelemetry.Instrumentation.GrpcNetClient`](https://www.nuget.org/packages/opentelemetry.instrumentation.grpcnetclient)
+package. Also, add any other instrumentations & exporters you will need.
 
 ```shell
 dotnet add package OpenTelemetry.Instrumentation.GrpcNetClient
 ```
 
+### Step 2: Enable Grpc.Net.Client Instrumentation at application startup
+
+Grpc.Net.Client instrumentation must be enabled in your code. The following
+example demonstrates how to do this for an ASP.NET Core application. The
+example also sets up the OpenTelemetry Console exporter and ASP.NET Core
+instrumentation which require adding additional packages the following
+additional packages:
+
+* [`OpenTelemetry.Instrumentation.AspNetCore`](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNetCore)
```

Comment:
Minor: consider linking to the package README file in the repo, as it gives benefits:
1. for folks who want to read more and do customization
2. for CI to check broken links

Reply:
Yes, I like this. @cijothomas, the ASP.NET doc has similar words. Do you think it makes sense to adopt this as our standard when referring to OpenTelemetry packages from another package's readme? That is, instead of a link to nuget.org a link to the respective readme inside the repo.